plugins/modules/remote_management/hpilo/hpilo_info.py

```diff
@@ -122,6 +122,226 @@
     type: str
     sample: ON
     version_added: 3.5.0
+
+server_name:
+    description: Server name
+    returned: always
+    type: str
+    sample: milo.company.com
+
+snmp_im_settings:
+    description: System serial number
+    returned: always
+    type: dict
```

Comment:
If the format of the contents is known, you should add `contains:` and describe the fields in it as well.